lwk_wollet/src/clients/electrum_client.rs

```diff
@@ -113,7 +119,10 @@ impl Debug for ElectrumClient {
 
 #[derive(Default)]
 pub struct ElectrumOptions {
-    timeout: Option<u8>,
+    pub timeout: Option<u8>,
+    pub retry: u8,
```

Comment:
This as default would be 0, but in `electrum_client::Config` the default is 1 (not sure what happens with 0)